### PR TITLE
add example for grpc

### PR DESCRIPTION
**Pull request by ilackarms**
Signed-off-by: ilackarms <sdw35@cornell.edu>

### CODE REVIEW

**Comment by ilackarms** [comment]
@yuval-k added a readme for a quick grpc demo

File: example/grpc/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+
+## Try the gRPC Service directly with cURL
+
+```
+curl $GRPC_URL
+curl $GRPC_URL/bookstore.Bookstore/ListShelves
+```
+
+# Use Gloo to translate cURL to gRPC
```
Comment (yuval-k) [inline example/grpc/README.md]: by cURL you mean json?

File: example/grpc/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+GRPC_ADDR=$(kubectl get po -l app=grpc-bookstore -n default -o 'jsonpath={.items[0].status.hostIP}'):$(kubectl get svc grpc-bookstore -n default -o 'jsonpath={.spec.ports[0].nodePort}')
+export GRPC_URL=http://$GRPC_ADDR
+```
+
+
+## Try the gRPC Service directly with cURL
+
+```
+curl $GRPC_URL
+curl $GRPC_URL/bookstore.Bookstore/ListShelves
```
Comment (yuval-k) [inline example/grpc/README.md]: is this thing going to work? wouldn't this return a protobuf?
if not we should let the user know the expected result

File: example/grpc/README.md (hunk 1 above)
Comment (ilackarms) [reply example/grpc/README.md]: maybe :stuck_out_tongue_winking_eye:

**Pull request merged**